### PR TITLE
Add `targetRanges` to |InputEventInit| and fixes "&lt;" issue

### LINKED ISSUE #47: Add `sequence<StaticRange> targetRanges` to `InputEventInit`
Since `InputEvent` has method `getTargetRanges()`, it would be reasonable to add
```
sequence<StaticRange> targetRanges = [];
```
to `InputEventInit`.

---

Also, can we change
```
"sequence getTargetRanges();" => "sequence<StaticRange> getTargetRanges();"
```

### PR DESCRIPTION
**Pull request by chong-z**
This PR fixes #47.

### CODE REVIEW

**Comment by chong-z** [comment]
Marked as non-substantive for IPR from ash-nazg.

File: index.html [hunk 1]
```diff
@@ -187,12 +187,13 @@
                 partial interface InputEvent {
                     readonly attribute DOMString     inputType;
                     readonly attribute DataTransfer? dataTransfer;
-                    sequence<StaticRange> getTargetRanges();
+                    sequence&lt;StaticRange&gt; getTargetRanges();
```
Comment (johanneswilm) [inline index.html:190]: @choniong So this line is reverting the change @foolip proposed a few days ago. Do you two agree on this change?

Comment (johanneswilm) [reply index.html:190]: Ah sorry, I see now it was likely me misunderstanding what foolip was asking for.

**Comment by foolip** [comment]
This is all valid WebIDL, and people will understand what's intended, but it's actually missing something connecting the `targetRanges` member of the dictionary to the `getTargetRanges()` method of the interface. I have no idea what these ranges are, but perhaps you'll want an internal slot on the event object, set by the constructor and read by the `getTargetRanges()` method.

Or, just use `[SameObject] readonly attribute FrozenArray<StaticRange> targetRanges` on the interface, that'd be more in line with how events and their init dicts usually work.

**Pull request merged**